command/server.go

```diff
@@ -941,12 +941,14 @@ func (c *ServerCommand) Run(args []string) int {
 	}
 
 	inmemMetrics, metricSink, prometheusEnabled, err := configutil.SetupTelemetry(&configutil.SetupTelemetryOpts{
-		Config:      config.Telemetry,
-		Ui:          c.UI,
-		ServiceName: "vault",
-		DisplayName: "Vault",
-		UserAgent:   useragent.String(),
-		ClusterName: config.ClusterName,
+		Config:              config.Telemetry,
+		Ui:                  c.UI,
+		ServiceName:         "vault",
+		DisplayName:         "Vault",
+		UserAgent:           useragent.String(),
+		ClusterName:         config.ClusterName,
+		MaxGaugeCardinality: config.Telemetry.MaximumGaugeCardinality,
+		GaugeInterval:       config.Telemetry.UsageGaugePeriod,
```

Comment:
It feels redundant to have the latter two in SetupTelemetryOpts when they are already in config.Telemetry? I think this is fine, but you could get rid of a few lines if you were inclined?

Reply:
Yeah, good point.